CONTRIBUTING.md

```diff
@@ -45,15 +45,88 @@ Bad:  Added some feature
 Bad:  Adding some feature
 Good: Add some feature
 ```
+### Running Locally
```

Comment:
You should probably mention that you need to be in the src directory for supermarket in order to do the bundle install

Reply:
It might also be worth mentioning that you should install the version of Ruby that Supermarket uses